packages/cli-optique/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@fedify/cli-optique",
```

Comment:
This should be renamed to `@fedify/cli` eventually (before merging this pull request).